source-code-manager/src/main/java/org/jboss/da/scm/impl/DirectoryReference.java

```diff
@@ -5,17 +5,21 @@
 import java.lang.ref.WeakReference;
 import java.time.Instant;
 import java.util.Optional;
+import javax.inject.Inject;
 import org.apache.commons.io.FileUtils;
+import org.jboss.da.scm.api.SCM;
 import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 /**
  *
  * @author Honza Brázdil <jbrazdil@redhat.com>
  */
 class DirectoryReference {
 
-    Logger log = LoggerFactory.getLogger(DirectoryReference.class);
+    @Inject
+    Logger log;
+
+    //Logger log = LoggerFactory.getLogger(DirectoryReference.class);
```

Comment:
Here must stay the LoggerFactory, becuse DirectoryReference is not a bean (the container won't inject anything into it).